.eslintrc.js

```diff
@@ -1,16 +1,16 @@
-module.exports = {
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    ecmaFeatures: {
-      ecmaVersion: 2020,
-      sourceType: 'module',
-    },
-  },
-  extends: ['plugin:@typescript-eslint/recommended', 'prettier', 'plugin:prettier/recommended'],
-  rules: {
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
-    '@typescript-eslint/no-empty-function': 'off',
-    '@typescript-eslint/ban-types': ['error', { extendDefaults: true, types: { '{}': false } }],
-    '@typescript-eslint/ban-ts-comment': 'off',
-  },
-};
+// module.exports = {
```

Comment:
Can this be removed instead of commented out in main?

Reply:
Yeah, will do.